core/src/com/unciv/ui/CivilopediaScreen.kt

```diff
@@ -88,11 +88,9 @@ class CivilopediaScreen(ruleset: Ruleset) : CameraStageBaseScreen() {
                 .map { CivilopediaEntry(it.name,it.getDescription(ruleset),
                         ImageGetter.getTechIconGroup(it.name,50f)) }
 
-        categoryToEntries["Tutorials"] = Tutorials().getTutorialsOfLanguage("English").keys
-                .filter { !it.startsWith("_") }
-                .map { CivilopediaEntry(it.replace("_"," "),
-                        Tutorials().getTutorials(it, UncivGame.Current.settings.language)
-                                .joinToString("\n\n")) }
+        tutorialMiner.getCivilopediaTutorials(UncivGame.Current.settings.language)
```

Comment:
Under what situation would this be null?

Reply:
Cool
There's no inherent problem with null if it's a real possibility, but if not then it causes a lot of .? and .let and other kotlin-specific operators that are not immediately readable for people from, say, a Java or C# background
As an open-source project, once if the things we try to maximize on is readability for people unfamiliar with the codebase